steps/infisical-secrets/0.0.2/step.yml

```diff
@@ -0,0 +1,65 @@
+title: Infisical Secrets
+summary: |
+  Pull secrets from an Infisical secrets repository
+description: |
+  Use this step to access your cloud or self hosted Infisical secrets and inject them into your build pipeline.
+website: https://github.com/CodeFishStudio/bitrise-step-infisical-secrets
+source_code_url: https://github.com/CodeFishStudio/bitrise-step-infisical-secrets
+support_url: https://github.com/CodeFishStudio/bitrise-step-infisical-secrets/issues
+published_at: 2024-08-21T16:31:30.683254+09:30
+source:
+  git: https://github.com/CodeFishStudio/infisical-bitrise-step.git
+  commit: ca5df8db8b033c5f7c0d77f3c37c68ae7b7e436a
```

Comment:
It seems the `0.0.2` tag points to the `[5cbb3a5](https://github.com/CodeFishStudio/infisical-bitrise-step/commit/5cbb3a5b02ae5939db1c49ac7f86bc983c225d4e)` commit in the repository. 
Please don't move Step version tags, after you share a Step version.